src/policy/fees.h

```diff
@@ -260,6 +261,9 @@ class CBlockPolicyEstimator
     /** Check stats are consistent (for unit tests) */
     int CheckConsistent() const EXCLUSIVE_LOCKS_REQUIRED(m_cs_fee_estimator);
 
+    /** Initialise buckets up to base */
+    size_t ExtendBucketsUpTo(double base, double min) EXCLUSIVE_LOCKS_REQUIRED(m_cs_fee_estimator);
```

Comment:
I would appreciate the documentation on this method to be a little bit more descriptive. IIUC it adds buckets on the lower end from min up to base?

Reply:
Update the comment